Basic/ClientDriven/Assets/Scripts/ServerIngredientPhysics.cs

```diff
@@ -14,13 +13,18 @@ public class ServerIngredientPhysics : NetworkBehaviour
 
     public override void OnNetworkObjectParentChanged(NetworkObject parentNetworkObject)
     {
-        SetPhysics(parentNetworkObject == null);
+        // The ingredient's physic needs to be disabled when the ingredient is parented so that it's following the parent instead of moving freely.
+        SetKinematic(parentNetworkObject != null);
     }
 
-    void SetPhysics(bool isEnabled)
+    private void SetKinematic(bool isEnabled)
```

Comment:
nitpick (no need to fix it): `private` is implicit in C#, we don't have to write it. In fact, by not writing it, codes is faster  to read (as there's less stuff) and it becomes easier to distinguish between public and private method with a glympse

Reply:
Oh, I think my rider setting just add them automatically, I'll fix that for the next PR, thanks!